.changeset/tall-buses-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/store": major
```

Comment:
is this a major change?

Reply:
kind of, I add an argument to `renderRecordData`, which is technically an exported function that somebody could be using, albeit unlikely

Reply:
since we tend to default this to `""` in a few places, we could make the argument optional with `'""` as the default and make it a minor change

Reply:
updated, don't have a strong opinion here and will trust ur judgement. Initially I'd considered this, but I'm kinda biased against optional args